src/main/java/core/basesyntax/DataConverterImpl.java

```diff
@@ -0,0 +1,33 @@
+package core.basesyntax;
+
+import java.util.ArrayList;
+import java.util.List;
+
+public class DataConverterImpl implements DataConverter {
+
+    @Override
+    public List<FruitTransaction> convertToTransaction(List<String> inputData) {
+        List<FruitTransaction> transactions = new ArrayList<>();
+        for (String line : inputData) {
+            if (line.startsWith("type")) {
+                continue;
+            }
+            String[] parts = line.split(",");
```

Comment:
Splitting the line by comma directly in the code is a sort of hardcoding. If the file format changes, you'll have to change this part of the code. Consider making the delimiter a constant or a configurable parameter.

Reply:
changes have been made